jwst/pipeline/calwebb_spec2.py

```diff
@@ -254,10 +268,7 @@ def process_exposure_product(
         # If there is only one `imprint` member, this imprint exposure is subtracted from all the
         # science and background exposures.  Otherwise, there will be as many `imprint` members as
         # there are science plus background members.
-
         calibrated = self.imprint_subtract(calibrated, members_by_type['imprint'])
-
-        # for each background image subtract an associated leak cal
```

Comment:
Why remove this comment? I think it's useful to let people know the purpose of lines 272-274 and how they differ from what was done to the science exposure in line 271.

Reply:
I'm putting it back because you're right that its removal is extraneous to this PR, but IMO this comment is redundant with the code itself.  It's obvious that the code is looping over background files and doing the imprint subtraction on the background files